app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1416,4 +1421,177 @@ class PatientRepositoryAndroidTest {
     assertThat(patientRepository.findPatientWithBusinessId(deletedBusinessIdentifier).blockingFirst()).isEqualTo(None)
     assertThat(patientRepository.findPatientWithBusinessId("missing_identifier").blockingFirst()).isEqualTo(None)
   }
+  
+  @Test
+  fun if_patient_is_defaulter_then_isPatientDefaulter_should_return_true() {
+
```

Comment:
Extra line?

Reply:
This is added because there  `fun` here.
I found it more readable this way.